### PR TITLE
Add Contact Section to Portfolio

### PR DESCRIPTION
**Pull request by frempongdev**
In this Pull Request I made the following Changes : 

- Added Contact Section. 
- - Updated HTML with section for contacts.
- - Pair with Coding partner to add Forms. 
- - Style forms to meet requirements.
- - Corrected all Linter Errors.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -387,4 +398,78 @@
...
+
+  .forms-section {
+    background-color: #6070ff;
+    background-image: url(assets/mobile/contact\ background.png);
+    background-size: cover;
+    background-repeat: no-repeat;
+    border-top-left-radius: 70px;
+    position: relative;
+    top: -80px;
+  }
```
Comment (leonardodiasb) [inline style.css:412]: - [ ] Your form style is great, but the `top: -80px;` attribute is creating whitespace at the bottom of the page. Kindly fix this issue and remove the whitespace to make your page looks more professional and similar to the Figmat template. (by default the `body` element has a margin on some browsers, it is nice to set the margin of the `body` element to 0 so the design looks consistent)
 - Your design:
 - 
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @frempongdev, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Professional PR. ✔️
 - No linter errors. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (frempongdev) [reply style.css]: Fixed 😄 🚀

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @frempongdev 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub-Flow.
✔️ Professional README  
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**